<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>THPC</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QGroupBox" name="sensor_1" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title" >
     <string>Sensor  1</string>
    </property>
    <widget class="QLCDNumber" name="temp_1" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="smallDecimalPoint" >
      <bool>true</bool>
     </property>
     <property name="segmentStyle" >
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0" >
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLCDNumber" name="humi_1" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>140</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="smallDecimalPoint" >
      <bool>true</bool>
     </property>
     <property name="segmentStyle" >
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3" >
     <property name="geometry" >
      <rect>
       <x>130</x>
       <y>10</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Temperature °C</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4" >
     <property name="geometry" >
      <rect>
       <x>130</x>
       <y>120</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Humidity % rel.</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="sensor_2" >
    <property name="geometry" >
     <rect>
      <x>240</x>
      <y>10</y>
      <width>221</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title" >
     <string>Sensor 2</string>
    </property>
    <widget class="QLCDNumber" name="humi_2" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>140</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="smallDecimalPoint" >
      <bool>true</bool>
     </property>
     <property name="segmentStyle" >
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="temp_2" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="smallDecimalPoint" >
      <bool>true</bool>
     </property>
     <property name="segmentStyle" >
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label" >
     <property name="geometry" >
      <rect>
       <x>130</x>
       <y>120</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Humidity % rel.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2" >
     <property name="geometry" >
      <rect>
       <x>125</x>
       <y>10</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Temperature °C</string>
     </property>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="clock" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>260</y>
      <width>441</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Plain</enum>
    </property>
    <property name="numDigits" >
     <number>8</number>
    </property>
    <property name="segmentStyle" >
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="sensor_3" >
    <property name="geometry" >
     <rect>
      <x>470</x>
      <y>10</y>
      <width>261</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title" >
     <string>Pressure</string>
    </property>
    <widget class="QLCDNumber" name="buffer_pres" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>140</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="smallDecimalPoint" >
      <bool>true</bool>
     </property>
     <property name="segmentStyle" >
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="ambient_pres" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="smallDecimalPoint" >
      <bool>true</bool>
     </property>
     <property name="segmentStyle" >
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QProgressBar" name="useable_pres" >
     <property name="geometry" >
      <rect>
       <x>220</x>
       <y>20</y>
       <width>31</width>
       <height>311</height>
      </rect>
     </property>
     <property name="maximum" >
      <number>1</number>
     </property>
     <property name="value" >
      <number>1</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_5" >
     <property name="geometry" >
      <rect>
       <x>76</x>
       <y>10</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>ambient pressure mbara</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6" >
     <property name="geometry" >
      <rect>
       <x>88</x>
       <y>120</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>buffer pressure mbara</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
